packages/tpetra/core/src/Tpetra_CrsGraph_def.hpp

```diff
@@ -5724,7 +5727,7 @@ namespace Tpetra {
     Kokkos::deep_copy(row_ptrs_beg, this->k_rowPtrs_);
 
     const size_t N = (row_ptrs_beg.extent(0) == 0 ? 0 : row_ptrs_beg.extent(0) - 1);
-      row_ptrs_type row_ptrs_end("row_ptrs_end", N);
+    row_ptrs_type row_ptrs_end("row_ptrs_end", N);
```

Comment:
In the future, let's allocate this Kokkos::View without initializing, if that makes sense.